### PR TITLE
Bump version to current in getting started pages

### PR DESCRIPTION
**Pull request by GrantJamesPowell**
It seems the docs page is slightly behind the published version

### CODE REVIEW

**Comment by maennchen** [comment]
@GrantJamesPowell Thanks for the PR!

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Parse Cron Format Strings, Write Cron Format Strings and Calculate Execution Dat
 
 ```elixir
 def deps do
-  [{:crontab, "~> 1.1.2"}]
+  [{:crontab, "~> 1.1.10"}]
```
Comment (maennchen) [inline README.md:16]:
```suggestion
  [{:crontab, "~> 1.1"}]
```

File: pages/Getting Started.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
     ```elixir
     def deps do
-      [{:crontab, "~> 0.8.5"}]
+      [{:crontab, "~> 1.1.10"}]
```
Comment (maennchen) [inline pages/Getting Started.md:8]:
```suggestion
      [{:crontab, "~> 1.1"}]
```

**Review by maennchen — CHANGES_REQUESTED** [review]
I think we should loosen the recommended requirements a bit at the same time.

**Comment by GrantJamesPowell** [comment]
Glad to help! Super helpful library!

**Review by maennchen — APPROVED** [review]

**Pull request merged**